### ISSUE TITLE
Invalid Database Errors with Additional Keys When Connecting to Database via Gateway

### ISSUE THREAD

**Issue by np-daxton** [azimuttapp/azimutt#302]
Hello Azimutt Team,

I've encountered a couple of database validation errors when trying to connect my local database through the self-deployed Azimutt Gateway. Below are the errors:

1. **Connecting to Local MySQL Error:**
```
Invalid Database, 14 issues found in 1 group:
- at .entities.?.attrs.?: invalid additional key 'values' (["bar","barStack","hbar","hbarStack","line","lineArea","pie"]) (.entities.1.attrs.2 and 13 more)
```
![image](https://github.com/azimuttapp/azimutt/assets/162109483/31d0295b-0b64-4f56-988f-d0057873e60f)

2. **Connecting to Local Docker MySQL Error:**
```
Invalid Database, 2 issues:
- at .entities.76.attrs.23: invalid additional key 'values' (["STATEMENT"])
- at .entities.77.attrs.23: invalid additional key 'values' (["STATEMENT"])
```
![image](https://github.com/azimuttapp/azimutt/assets/162109483/7fcbbe16-e9b6-4399-bacc-350d5934793f)

These errors suggest that there are invalid additional keys in the entity attributes which are not expected by the Azimutt schema validation.

### Details:
- **Deployment Method:** Self-hosted Azimutt Gateway
- **Database Type:** MySQL 8.0.36

### Questions:
1. Could you provide guidance on what might be causing these 'invalid additional key' errors?
2. Is there a specific configuration or database schema requirement that I might be missing?

I appreciate any help or insights you can provide as I am keen to resolve these issues and successfully connect my database through the Gateway.

Thank you for your support!

**Comment by loicknuchel** [comment]
Hi,

I answered you by email but put the same answer here:
Can you check you run the last version of the Gateway (1.7.0 currently).
The Docker image automatically upgrades as there is no version number but that's not the case for the gateway.
My guess is you have a mismatch between both versions of the code.

Tell me how it goes.

Cheers!

**Comment by np-daxton** [comment]
Hello,

Thank you for your prompt response and the suggestions.

Yes, I am using the Azimutt Docker Image tagged as ghcr.io/azimuttapp/azimutt:main. Regarding the Gateway version, I am running it with Node.js version 18.19.1, and the Gateway version is 0.1.7. I have tried running it both from the GitHub source code and using the command npx azimutt@latest gateway, and in both cases, the version is 0.1.7.

Please let me know if there are any additional steps I should take or if there's a newer version of the Gateway I should be using.
![image](https://github.com/azimuttapp/azimutt/assets/162109483/a7fe59a8-e65b-4802-a261-a382ca92e115)

Thank you once again for your help!

**Comment by loicknuchel** [comment]
You were totally right, the `values` attribute was wrongly added.
I fixed it in the `0.1.8` version, you can launch it with the latest tag: `npx azimutt@latest gateway`
Tell me how it goes.

**Comment by np-daxton** [comment]
Hello,

I've tested the updated version 0.1.8 and I'm happy to report that everything is now working perfectly. Thank you for the swift response and resolution!
